app/assets/stylesheets/users.scss

```diff
@@ -351,12 +359,13 @@
 
 .users-circuits-tab {
   background-color: $form-border-grey;
-  border-radius: 0;
+  border-radius: 5px;
   color: $black;
   font-size: 18px;
-  padding: 7px 25px;
-  margin: 0;
+  margin-left: 7px;
+  margin-right: 7px;
```

Comment:
For gap, avoid adding margin directly to button.
Instead, add `gap` in the parent class `users-circuits-navbar-buttons`

Reply:
Done sir.